images/cilium/Dockerfile

```diff
@@ -10,14 +10,14 @@ ARG NOSTRIP
 ARG LOCKDEBUG
```

Comment:
What happened to the `ARG V`?

Reply:
Context: `ARG V` allows developers to specify a verbosity flag where for instance I define it to `V=0` in my shell to cause the build system to be less verbose. This allows devs to get a clearer picture of what fails, while still printing any and all possibly relevant information during builds on automated systems where you may need to introspect later.

Reply:
I agree that control of verbosity can useful, but GitHub Actions hides logs for each step, so there is no reason to worry about build log verbosity.
Secondly, when building locally BuildKit only presents per-step logs on failure, which is very different from old `docker build` output format.
Personally, I don't think there is a better way than passing `V` from make to docker, and back to make, so I dropped `V` for now, as this PR is focus only on enabling builds in GitHub Actions. I'd rather provide high-level flags as part of a higher-level tool (cilium/image-tools#10), which would also enable a way of building all of the images more consistently and avoid the boilerplate.